functions/export-to-csv/2.0/index.js

```diff
@@ -0,0 +1,30 @@
+import { ExportToCsv } from 'export-to-csv';
+const exportToCSV = async ({
+  collectionInput,
+  delimiter,
+  model: { name: modelName },
+  property: [{ name: propertyName }],
+}) => {
+  return {
+    reference: await storeFile(modelName, propertyName, {
+      contentType: 'text/csv',
+      extension: 'csv',
+      fileName: `${modelName}-export`,
+      fileBuffer: stringToBuffer(
+        new ExportToCsv({
+          fieldSeparator: delimiter,
+          quoteStrings: '"',
+          decimalSeparator: '.',
+          showLabels: true,
+          showTitle: false,
+          title: 'Diverted Documents Search Export ',
+          useTextFile: false,
+          useBom: true,
+          useKeysAsHeaders: true,
```

Comment:
We could add all these fields to the options. That would mean that we, team actions, have to provide a checkbox option.

Reply:
@thomasvanophem I believe there's alreadya switch available (not a checkbox) but it does the same, I can add these into the `function.json` :)

Reply:
@thomasvanophem I've added the options to `functions.json` but now some options like the titleName will still be shown even though the useTitle is disabled.